sources/niUI/src/API/niUI/IWidgetTree.h

```diff
@@ -238,6 +238,12 @@ struct iWidgetTreeNode : public iUnknown
   //! Get the node's name.
   //! {Property}
   virtual const achar* __stdcall GetName() const = 0;
+  //! Set the node's display name.
+  //! {Property}
+  virtual tBool __stdcall SetDisplayName(const achar* aVal) = 0;
+  //! Get the node's display name.
+  //! {Property}
+  virtual const achar* __stdcall GetDisplayName() const = 0;
```

Comment:
Looks reasonable, I assume this is so that the display name can be different from the "internal name". I just want to make sure its not used only for localisation. For for localisation we should go through iHString::GetLocalized().

@ArantirZhang can you tell me what this is used for?